### ISSUE TITLE
Data file corrupted

### ISSUE THREAD

**Issue by leowyy99** [leowyy99/pe#6]
`list meals all` command does not work if the data file is corrupted with wrong data

before:

![image.png](https://raw.githubusercontent.com/leowyy99/pe/main/files/0d82e6f5-4b66-4d9a-91ab-309da527d860.png)

after:

![image.png](https://raw.githubusercontent.com/leowyy99/pe/main/files/919c27c0-0eb7-4d27-b0ea-be5d35d7b8b6.png)

![image.png](https://raw.githubusercontent.com/leowyy99/pe/main/files/8874f077-a138-48af-bd6e-9ec24ecfa8b6.png)

Basically program does not handle corrupted data files, and fail to throw correct exceptions.
